client/src/pages/Home.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import Container from '@material-ui/core/Container';
+import Login from './Login';
+import Signup from './Signup';
+
+const Home = () => {
+	const [ page, setPage ] = useState('login');
+
+	const handleClick = (location) => {
+		setPage(location);
+	};
+
+	let Content;
+
+	if (page === 'login') {
+		Content = <Login handleClick={handleClick} />;
+	} else {
+		Content = <Signup handleClick={handleClick} />;
+	}
```

Comment:
Instead of handling the Login / Signup pages in state, you are going to want to use React Router. The reason is because React Router will do a lot of things for you for free, such as handling browser history. If you use state, it'll also be more code and more maintenance for you

Reply:
Thank you. I will add this on the next PR.